app/views/miq_ae_customization/_dialog_field_form.html.haml

```diff
@@ -1,274 +1,384 @@
-                  "data-miq_observe" => {:interval => '.5', :url => url}.to_json,
-                  :disabled          => @edit[:field_validator_type].blank?,
-                  :maxlength         => 250)
-                \/
-
-        - elsif %w(DialogFieldDateControl DialogFieldDateTimeControl).include?(@edit[:field_typ])
-          %tr
-            %td.key= _('Read only')
-            %td
-              = check_box_tag('field_read_only', '1', @edit[:field_read_only], "data-miq_observe_checkbox" => {:url => url}.to_json)
+                = text_field_tag("field_default_value",
+                                  @edit[:field_default_value],
+                                  "data-miq_observe" => {:interval => '.5', :url => url}.to_json,
+                                  :class => "form-control",
+                                  :maxlength         => 50)
+            - else
+              = check_box_tag("field_default_value", "1",
+                              @edit[:field_default_value],
+                              "data-miq_observe_checkbox" => {:url => url}.to_json)
+        - if @edit[:field_typ].include?('TextBox')
+          .form-group
+            %label.col-md-2.control-label
+              = _('Protected')
+            .col-md-8
+              = check_box_tag('field_protected', 'true',
+                              @edit[:field_protected],
+                              "data-miq_observe_checkbox" => {:url => url}.to_json)
+        .form-group
+          %label.col-md-2.control-label= _('Required')
+          .col-md-8
+            = check_box_tag('field_required', 'true',
+                            @edit[:field_required],
+                            "data-miq_observe_checkbox" => {:url => url}.to_json)
+        .form-group
+          %label.col-md-2.control-label= _('Read only')
+          .col-md-8
+            = check_box_tag('field_read_only', '1', @edit[:field_read_only],
+                            "data-miq_observe_checkbox" => {:url => url}.to_json)
+        .form-group
+          %label.col-md-2.control-label
+            = _('Auto Refresh other fields when modified')
+          .col-md-8
+            = check_box_tag('field_trigger_auto_refresh', '1',
+                            @edit[:field_trigger_auto_refresh],
+                            'data-miq_observe_checkbox' => {:url => url}.to_json)
+        - if @edit[:field_typ].include?('TextBox')
+          .form-group
+            %label.col-md-2.control-label
+              = _('Validator Type')
+            .col-md-8
+              = select_tag('field_validator_type',
+                            options_for_select([[_("None"), nil], [_("Regular Expression"), 'regex']],
+                                                @edit[:field_validator_type]),
+                            "data-miq_sparkle_on" => true,
+                            :class    => "selectpicker")
+            :javascript
+              miqInitSelectPicker();
```

Comment:
@PanSpagetka only need single call to miqInitSelectPicker in a file. Also curious if observers are working for checkbox fields or do we have to do something special to send up checkbox transactions after bootstrap conversion.